package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "zonos-typescript-sdk",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "graphql": "^16.6.0",
+    "graphql-request": "^4.3.0",
+    "graphql-tag": "^2.12.6"
+  },
+  "engines": {
+    "node": "18"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.12",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-typescript": "^7.21.4",
+    "@graphql-codegen/add": "^3.2.3",
+    "@graphql-codegen/cli": "^2.14.1",
+    "@graphql-codegen/introspection": "^2.2.1",
+    "@graphql-codegen/typescript": "^2.8.8",
+    "@graphql-codegen/typescript-graphql-request": "^4.5.9",
+    "@graphql-codegen/typescript-operations": "^2.5.7",
+    "@rollup/plugin-buble": "^1.0.2",
+    "@rollup/plugin-node-resolve": "^15.0.1",
+    "@rollup/plugin-terser": "^0.4.0",
+    "@types/jest": "^29.2.5",
+    "@types/node": "^18.11.18",
+    "@typescript-eslint/eslint-plugin": "^5.57.0",
+    "@typescript-eslint/parser": "^5.57.0",
+    "concurrently": "^7.4.0",
+    "eslint": "^8.37.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-deprecation": "^1.3.3",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.2.1",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
+    "glob": "^9.3.2",
+    "jest": "^29.5.0",
+    "prettier": "^2.8.7",
+    "rollup": "^3.20.2",
+    "rollup-plugin-progress": "^1.1.2",
+    "rollup-plugin-sizes": "^1.0.5",
+    "rollup-plugin-typescript2": "^0.34.1",
+    "ts-jest": "^29.0.5",
+    "ts-node": "^10.9.1",
+    "ts-toolbelt": "^9.6.0",
+    "tsconfig-paths-webpack-plugin": "^4.0.1",
+    "ttypescript": "^1.5.15",
+    "typescript": "^4.9.5",
```

Comment:
Hmmm, there is a newer version of typescript now, but I know from experience that the `ttypescript` doesn't work yet with it. I wonder if we can find another option. Some quick digging: https://github.com/cevek/ttypescript/issues/140#issuecomment-1488575848

Reply:
Yah, at first I upgrade to typescript 5, but I downgraded back to 4.9.5. And that's the reason why 😢 
I'm not sure what will be the alternative yet, but I will do more research on it.

Reply:
Even if there isn't anything ready yet. I think it's worth keeping up to date to keep up with newer versions of typescript.

Reply:
Yes, I agree, but there isn't a version of `ttypescript` or `ts-patch` that works with typescript 5 yet. I'll try to upgrade when there is one. Thanks